plugins/chat/app/services/chat/publisher.rb

```diff
@@ -129,7 +129,14 @@ def self.user_tracking_state_message_bus_channel(user_id)
     end
 
     def self.publish_user_tracking_state(user, chat_channel_id, chat_message_id)
-      data = { channel_id: chat_channel_id, last_read_message_id: chat_message_id }.merge(
+      data = {
+        channel_id: chat_channel_id,
+        last_read_message_id: chat_message_id,
+        # TODO (martin) Remove old chat_channel_id and chat_message_id keys here once deploys have cycled,
+        # this will prevent JS errors from clients that are looking for the old payload.
```

Comment:
Wanna put this in a topic with a bookmark timer?

Reply:
Yah was going to put it in mine :)